CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ set(PYBIND11_HEADERS
   include/pybind11/descr.h
   include/pybind11/options.h
   include/pybind11/eigen.h
+  include/pybind11/embedded.h
```

Comment:
very superficial: I would prefer a verb or a noun for the header file name (e.g. ``embed.h``)